utils/telemetry/events.go

```diff
@@ -0,0 +1,37 @@
+package telemetry
+
+import (
+	"time"
+)
+
+// REALMC-7243 same as the new cli Segment tracking
+
+type event struct {
+	id          string
+	eventType   EventType
+	userID      string
+	time        time.Time
+	executionID string
+	command     string
+	data        []EventData
+}
+
+type EventData struct {
+	Key   string
+	Value interface{}
+}
+
+type EventType string
+
+const (
+	EventTypeCommandStart EventType = "COMMAND_START"
+	EventTypeCommandEnd   EventType = "COMMAND_END"
+	EventTypeCommandError EventType = "COMMAND_ERROR"
+)
+
+const (
+	eventDataKeyCommand     = "command"
+	eventDataKeyExecutionID = "xid"
```

Comment:
can we include a `version`/`v` field (check how we're doing it on `master` now, it was a recent change/update)

Reply:
can we match these keys to what's in `master`?  looks like instead of `command` we use `cmd`, and instead of `version` we use `v`